src/v2/components/Billing/components/MyGroups/components/MyGroup/components/UpgradeSelection/index.js

```diff
@@ -28,7 +28,7 @@ const OptionLabel = styled(Text).attrs({
 
 const Option = styled(Box)`
   &:hover ${OptionLabel} {
-    color: black;
+    color: ${props => props.theme.colors.gray.bold};
```

Comment:
Incase you wanna save yourself some typing in the future: https://github.com/styled-system/styled-system/tree/master/packages/theme-get